### PR TITLE
Store data in javascript object to create the projects section dynamically, and create the popup when the  see-more button is clicked

### PR DESCRIPTION
**Pull request by desorgui**
## What?
Implementation of the project section dynamically with data stored in Javascript Object and the popup window when the see-more button is clicked.

## Why?
These changes complete the project requirements.

## How?
I used Html, and CSS (Grid and Flexbox) to be able to place each element of my project, and also change the display for mobile,  and Javascript to store data, create dynamically the project section, and create the popup window for each project.

## Testing?
I've added Linter for testing all new codes.

## Anything Else?
I used [template 2](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27) of the Microverse students' projects to implement my portfolio

### CODE REVIEW

File: custom.js [hunk 1]
```diff
@@ -2,6 +2,172 @@ const hamburger = document.querySelector('#nav-button');
...
+    WorkData[index].technologies.forEach((element) => {
+      const listItem = document.createElement('li');
+      listItem.className = 'itemCat';
+      listItem.textContent = element;
+      listContainer.appendChild(listItem);
+    });
+
+    popup.innerHTML += `
+    <div class="d-flex">
+            <div class="popup-image">
```
Comment (gtekle) [inline custom.js:146]: - [ ] The background color of the popup window (modal) is white in the Figma design. Kindly rectify.
- [ ] In the desktop version, the height of the pop-up window is short as compared to the Figma design. As a result the project snapshot images' width Vs height proportion is distorted. Kindly rectify.

#### Design (Desktop)
popup-window-desktop-figma
#### Impl (Desktop)
popup-window-desktop-impl

- [ ] In the mobile version, bottom part of the pop-up window (modal) is not visible. Kindly stick to the figma design.
#### design (Mobile)
popup-window-mobil-figma

#### Impl (Mobile)
popup-window-mobil-impl

**Review by gtekle — CHANGES_REQUESTED** [review]
Hi @desorgui :wave: , 

## STATUS: CHANGES REQUIRED

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
this-close


## Highlights
:heavy_check_mark: Meaningful commit messages
:heavy_check_mark: Clean code
:heavy_check_mark: Popup windows generated dynamically :+1: 

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly check the inline comments below.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [Optional] What do you think if you add  links that help users navigate to previous project and next project from the pop-up window? But, kindly remember that this is optional requirement.
prev-next-nav-feature


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by phelian23 — COMMENTED** [review]
Hi @desorgui, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is not passing some of the linters check. Kindly fix it and re-submit

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

**Review by gtekle — APPROVED** [review]
Hi @desorgui :wave: , 

# STATUS: APPROVED :tada: :tada: :tada: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
approved-stamp

## Highlights
:heavy_check_mark: Clean code
:heavy_check_mark: Nice UI

## Optional suggestions

- [ optional ] The images are not rendered as expected when I open your project online. This may affect you when you deployed to Github pages or somewhere else. Please take a look at [this](https://raw.githack.com/desorgui/portfolio/popup-window/index.html) link.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**